deployments/goerli/weth/deploy.ts

```diff
@@ -0,0 +1,160 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { debug, DeploySpec, deployComet, exp, sameAddress, wait } from '../../../src/deploy';
+
+const clone = {
+  cbETHImpl: '0x31724cA0C982A31fbb5C57f4217AB585271fc9a5',
+  cbETHProxy: '0xBe9895146f7AF43049ca1c1AE358B0541Ea49704',
+};
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const deployed = await deployContracts(deploymentManager, deploySpec);
+  await mintTokens(deploymentManager);
+  return deployed;
+}
+
+async function deployContracts(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const ethers = deploymentManager.hre.ethers;
+  const signer = await deploymentManager.getSigner();
+
+  // Declare existing assets as aliases
+  const WETH = await deploymentManager.existing('WETH', '0x42a71137C09AE83D8d05974960fd607d40033499', 'goerli');
+  const wstETH = await deploymentManager.existing('wstETH', '0x4942BBAf745f235e525BAff49D31450810EDed5b', 'goerli');
+
+  // Import shared contracts from cUSDCv3
+  // Note: We need to first spider the contracts on the cUSDC base. Consider moving this spider logic into `fromDep`.
```

Comment:
In the future we could also have a `fromDep` to return all aliases as object and could destructure ```
const { 
 cometAdmin, 
 cometFactory, 
 'configurator:implementation': $configuratorImpl,
  ...
  } = await deploymentManager.fromDep(network, base)``` and spider from there

Reply:
Eh, this breaks putAlias, which is why it is the way it is currently